libr/core/cmd.c

```diff
@@ -3148,6 +3148,7 @@ R_API void r_core_cmd_init(RCore *core) {
 	r_cmd_add (core->rcmd, "info",     "get file info", &cmd_info);
 	r_cmd_add (core->rcmd, "cmp",      "compare memory", &cmd_cmp);
 	r_cmd_add (core->rcmd, "seek",     "seek to an offset", &cmd_seek);
```

Comment:
Maybe we should have a struct defining all commands, descripttion, callbacks and constructors and just do a loop